src/sidebar/host-config.js

```diff
@@ -50,13 +56,33 @@ export default function hostPageConfig(window) {
     'usernameUrl',
   ];
 
+  const coercions = {
```

Comment:
As a follow-up to this PR, I think it would make sense to merge this object with the `paramWhiteList` above, such that there is an object whose keys are allowed parameters and functions are coercion or validation functions.

Reply:
good idea

Reply:
https://github.com/hypothesis/client/issues/1968